### ISSUE TITLE
Accessibility: Combobox: No focus indication provided for combo boxes, when accessed. 

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3458]
Description: No focus indication provided for combo boxes, when accessed. 

Expected: As per the ariba requirement focus indication should be provided for each & every UI when accessed

Issue screenshot:

![image](https://user-images.githubusercontent.com/32538291/94523556-fe2d0500-024e-11eb-9a2c-e46ecf916e67.png)
![image](https://user-images.githubusercontent.com/32538291/94523581-05541300-024f-11eb-98c3-82778f68b958.png)

Expected:

![image](https://user-images.githubusercontent.com/32538291/94523860-73003f00-024f-11eb-8641-ad1fd8253a97.png)

**Comment by mykolavorobiov** [comment]
PR with fix https://github.com/SAP/fundamental-ngx/pull/4204
